x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -82,6 +82,24 @@ query = '''
 process where match(command_line, '.*?net[1]?  localgroup.*?', '.*? myappserver.py .*?')
 '''
 
+[[queries]]
+expected_event_ids  = [50, 97, 98]
+query = '''
+process where match('.*?net[1]?  localgroup.*?', command_line,'.*? myappserver.py .*?')
```

Comment:
I don't exactly know the issue that motivated some of these changes, and am not sure why loosening up the signature of match is a good thing.

For this example, I'm confused how it's supposed to be evaluated. Which argument is the source string and which are the patterns to be matched against?